src/Components/Contact Us/Dropdown.jsx

```diff
@@ -0,0 +1,34 @@
+import Countries from "./Countries";
+
+const Dropdown = ({ selectedCounty, handleCountyChange }) => {
```

Comment:
what is the use of "selectedCounty" here if we are redefining "selectedCount*r*y" again inside the func?

Reply:
the variable was named wrong. it should have been named newCountry. the intended functionality was to preserve the state between renders.